cmd/juju/controller/addmodel_test.go

```diff
@@ -42,11 +43,15 @@ var _ = gc.Suite(&AddModelSuite{})
 
 func (s *AddModelSuite) SetUpTest(c *gc.C) {
 	s.FakeJujuXDGDataHomeSuite.SetUpTest(c)
+
+	agentVersion, err := version.Parse("2.2-rc1")
```

Comment:
Agree and I certainly intended for this version to be unrelated to any real Juju versions :D 
"2.55" in itself is invalid, I'll use "2.55.5". Here is hope we will not have a 2.55.5 release \o/